chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/ResponseValidatorProvider.java

```diff
@@ -0,0 +1,12 @@
+package com.griddynamics.jagger.engine.e1.collector;
+
+import com.griddynamics.jagger.engine.e1.scenario.KernelSideObjectProvider;
+
+/**
+ * Provides instances of a ${@link ResponseValidator} class
+ * @n
+ * Created by Andrey Badaev
+ * Date: 14/12/16
+ */
+public interface ResponseValidatorProvider extends KernelSideObjectProvider<ResponseValidator<Object,Object,Object>> {
```

Comment:
will the following validator work with this interface
```
...
public boolean validate(JHttpQuery<String> query, JHttpEndpoint endpoint, JHttpResponse<MetricEntity[]> result, long duration) {
        List<MetricEntity> actualEntities = Arrays.asList(result.getBody());
....
```

why not `interface ResponseValidatorProvider<Q, E, R> extends  KernelSideObjectProvider<ResponseValidator<Q, E, R>>`